### ISSUE TITLE
ImageColorToMask node returns a "mask" with values of 0 and 255, instead of values between 0 and 1

### ISSUE THREAD

**Issue by Borogravian** [comfyanonymous/ComfyUI#8763]
### Custom Node Testing

- [x] I have tried disabling custom nodes and the issue persists (see [how to disable custom nodes](https://docs.comfy.org/troubleshooting/custom-node-issues#step-1%3A-test-with-all-custom-nodes-disabled) if you need help)

### Expected Behavior

When using the node ImageColorToMask, I expected the output mask to be as per the documentation - a tensor with values clamped between 0 and 1. 

### Actual Behavior

Using "Preview Any", confirmed that output tensor actually has values of 255. 

### Steps to Reproduce

Use ImageColorToMask to create a "solid" mask and use Preview Any node to check output. 

### Debug Logs

```powershell
No debug logs
```

### Other

The problem code line is ComfyUI/comfy_extras/nodes_mask.py:155 :

`mask = torch.where(temp == color, 255, 0).float()`

Per the [mask documentation](https://docs.comfy.org/custom-nodes/backend/images_and_masks), that should read:

`mask = torch.where(temp == color, 1, 0).float()`

**Comment by comfyanonymous** [comment]
https://github.com/comfyanonymous/ComfyUI/commit/34c8eeec065856e835e5ccebfceb2ea3b76110a7
